src/views/pro-dashboard/ProDocuments.vue

```diff
@@ -320,6 +320,13 @@
       }),
       handleFilesChanged(files, type) {
         this.$data.documents[type] = [...this.$data.documents[type], ...files];
+
+        this.$toasts.toast({
```

Comment:
This is not true. If you upload the document without saving it, will not upload it to the server. So if you show the uploaded document toast message, and the user refreshes the page, he will be confused because he got the confirmation about his/her document being uploaded, and in fact nothing happened

Reply:
I wasn't sure about the exaclty behaviour so, make sense now. I'll change it.
Thx.